module/Application/src/Application/Assertion/HttpHostLocalhostAssertion.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Application\Assertion;
+
+use Zend\Permissions\Acl\Acl;
+use Zend\Permissions\Acl\Role\RoleInterface;
+use Zend\Permissions\Acl\Resource\ResourceInterface;
+use Zend\Permissions\Acl\Assertion\AssertionInterface;
+
+class HttpHostLocalhostAssertion implements AssertionInterface
```

Comment:
L'assertion non è più necessaria in quanto il dominio di provenienza non è un criterio di controllo degli accessi, ma di autenticazione